### ISSUE TITLE
package.json is missing. Yarn obligated?

### ISSUE THREAD

**Issue by rept** [getkuby/kuby-core#32]
I'm trying this out with a Rails 5.2 that isn't using Webpack.

When I'm building the image I get this at step 20:

```
Step 20/27 : COPY package.json .
COPY failed: stat /var/lib/docker/tmp/docker-builder133647128/package.json: no such file or directory
error: build failed: docker command exited with status code 1
```

Looking in the docs I found: 

    Yarn phase: Runs yarn install, which installs all the JavaScript dependencies listed in your app's package.json.

Is using Yarn a requirement for using Kuby or can I mark somewhere that this step isn't needed?

**Comment by rept** [comment]
I also tried this:

```
    docker do
      # Configure your Docker registry credentials here. Add them to your
      # Rails credentials file by running `bundle exec rake credentials:edit`.
      package_phase.remove('yarn')

      credentials do
       ...
```

but that doesn't seem to make any difference.

**Comment by rept** [comment]
With

      package_phase.remove(:yarn)

I see the step of installing yarn gets skipped but for some reason step 18 still remains

    Step 18/25 : COPY package.json .

Stange because I see that step

        dockerfile.copy('package.json', '.')

in the yarn_phase.rb file in the source?

**Comment by rept** [comment]
Yay!  Found it.

```
    docker do
      # Configure your Docker registry credentials here. Add them to your
      # Rails credentials file by running `bundle exec rake credentials:edit`.
      package_phase.remove(:yarn)
      delete :yarn_phase
```

Sorry for the spam.  Hopefully this helps someone in the future.

**Comment by camertron** [comment]
Hah, nice work @rept!

**Comment by JorgenIvarsson** [comment]
I think this is a big problem and that this issue needs to be opened again. The reason is that Rails 7 does not use yarn or webpacker by default. This "bug" will break the build phase for all new users who actually need this solution to deploy. It took me quite a while to find this solution. Thanks for your solution @rept !

**Comment by camertron** [comment]
Thanks @JorgenIvarsson, that makes sense. It's probably a good idea for Kuby to skip installing node, yarn, and JS deps if there's no package.json. I will add that fix to the upcoming release.

**Comment by camertron** [comment]
This has been addressed in the latest release, v0.18.0.
